### PR TITLE
feat: add phone call alert channel [gh-775]

### PR DESCRIPTION
**Pull request by maxigimenez**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
Add new `PhoneCallAlertChannel` construct for phone call alert channel

> Resolves #gh-775

### CODE REVIEW

**Comment by umutuzgur** [comment]
@maxigimenez is the BE already there?

**Comment by maxigimenez** [comment]
> @maxigimenez is the BE already there?

@umutuzgur Yes

File: packages/cli/src/constructs/phone-call-alert-channel.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { AlertChannel, AlertChannelProps } from './alert-channel'
+import { Session } from './project'
+
+export interface PhoneCallAlertChannelProps extends AlertChannelProps {
+  /**
+   * The phone number where to send the alert notifications.
```
Comment (umutuzgur) [inline packages/cli/src/constructs/phone-call-alert-channel.ts:6]: @maxigimenez we should put a link here to our docs which gives more info about which countries are supported

Comment (maxigimenez) [reply packages/cli/src/constructs/phone-call-alert-channel.ts:6]: @umutuzgur Good call. Done

**Review by umutuzgur — APPROVED** [review]

**Pull request merged**